gui/qt/i18n/fr_FR.ts

```diff
@@ -838,17 +838,17 @@
         <location filename="../mainwindow.ui" line="4859"/>
         <location filename="../mainwindow.cpp" line="573"/>
         <source>OS Stacks</source>
-        <translation>Stacks de l&apos;OS</translation>
+        <translation>Piles de l&apos;OS</translation>
```

Comment:
Compromis, histoire de :
```suggestion
        <translation>Piles (stacks) de l&apos;OS</translation>
```